tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -111,6 +111,18 @@ Verify Pytorch And Tensorflow Can Be Spawned
     Verify Image Can Be Spawned  image=pytorch  size=Default
     Verify Image Can Be Spawned  image=tensorflow  size=Default
 
+Verify That Blackbox-exporter Is Protected With Auth-proxy
+    [Documentation]     Verify that Blackbox-exporter is protected with auth-proxy and check it has 2 containers
+    [Tags]  Sanity
+    ...     Tier1
+    ...     ODS-1090
+    ${pod} =    Find First Pod By Name    namespace=redhat-ods-monitoring    pod_start_with=blackbox-exporter-
```

Comment:
Lines 119-124 should be in it's own keyword: `Verify BlackboxExporter Includes Oauth Proxy`